### PR TITLE
Improve integration instructions

### PR DESCRIPTION
**Pull request by lukasspiess**
This improves on almost all of the issues @yunzhouMSFT brought up. Thanks again!

### CODE REVIEW

**Comment by lukasspiess** [comment]
@scsouthw Can you maybe take a look and see if anything jumps out to you? Thanks.

**Comment by lukasspiess** [comment]
@scsouthw Should we add the support email address to our README?

**Comment by scsouthw** [comment]
Yes, let's use the support email address in the readme

File: README.md [hunk 1]
```diff
@@ -43,37 +43,39 @@ The SDK runs on devices with iOS 6.0 or higher.
...
-	[[MSAIAppInsights sharedInstance] setup];
-	// Do some additional configuration if needed here
-	...
-	[[MSAIAppInsights sharedInstance] start];
+	[MSAIAppInsights setup];
+	[MSAIAppInsights start];
 	```
 
-	You can also use the following shortcut:
+	If you want to do some additional configuration you have to call the sharedInstance directly between the `setup()` and `start()` calls:
```
Comment (scsouthw) [inline README.md]: Change to: 
"If additional configuration is needed, the sharedInstance can be accessed between the `setup()` and `start()` calls:

**Pull request merged**